modules/KalturaSupport/components/live/liveCore.js

```diff
@@ -95,6 +95,11 @@
 					vid.removeEventListener('progress', _this.onProgress);
 					_this.onProgress = null;
 				}
+                if (_this.offAirTimeout){
+                    clearTimeout(_this.offAirTimeout);
+                    _this.offAirTimeout = null;
+                }
+                clearInterval( _this.liveStreamStatusMonitor );
```

Comment:
use `removeLiveStreamStatusMonitor` api

Reply:
add a new api to reset `this.offAirTimeout` and use it here and in the rest places we reset it.